presto-yarn-package/src/main/slider/package/scripts/params.py

```diff
@@ -50,6 +50,7 @@
 coordinator_host = config['configurations']['global']['coordinator_host']
 presto_query_max_memory = config['configurations']['global']['presto_query_max_memory']
 presto_query_max_memory_per_node = config['configurations']['global']['presto_query_max_memory_per_node']
+presto_query_max_total_memory_per_node = config['configurations']['global']['presto_query_max_total_memory_per_node']
```

Comment:
this should be in separate commit, because it is not related to resource groups

Reply:
removed from this PR. @kokosing  please merge